tests/tracking/fluent/test_fluent.py

```diff
@@ -519,26 +520,81 @@ def test_get_run():
         assert run.info.user_id == "my_user_id"
 
 
+def call_search_runs():
+    return search_runs(output_format="list") if "MLFLOW_SKINNY" in os.environ else search_runs()
```

Comment:
Ideally, it'd be nice to test both output formats in the full client and skip the pandas output format tests in the skinny client. Perhaps parameterizing search_runs test cases by `output_format` and conditionally defining `output_format` based on `os.environ["MLFLOW_SKINNY"]` would help (via `@pytest.mark.parametrize`).